app/views/idv/how_to_verify/show.html.erb

```diff
@@ -1,17 +1,35 @@
 <% self.title = t('doc_auth.headings.how_to_verify') %>
-
 <%= render PageHeadingComponent.new.with_content(t('doc_auth.headings.how_to_verify')) %>
-<p id='how-to-verify-info'><%= t('doc_auth.info.how_to_verify') %></p>
-
+<%#   ============== Non Selfie Section ==========    %>
+<% if !@selfie_required %>
+  <p id='how-to-verify-info'><%= t('doc_auth.info.how_to_verify') %></p>
+<% else %>
+<%#   ============== Selfie Section ==========    %>
+<p id='how-to-verify-info'><%= t('doc_auth.info.how_to_verify_selfie') %></p>
+<% end %>
+<%#   ============== End Diff Section ==========    %>
```

Comment:
All these `if`/`else`s seem like a good use-case for a presenter class, so you could have:

```suggestion
<p id='how-to-verify-info'><%= @presenter.how_to_verify_info %></p>
```

And then...

```rb
class Idv::HowToVerifyPresenter
  # ...
  def how_to_verify_info
    if selfie_required
      t('doc_auth.info.how_to_verify_selfie')
    else
      t('doc_auth.info.how_to_verify')
    end
  end
end
```

Reply:
Good suggestion, thank you @aduth ! I wrote tests yesterday to test the conditional rendering but have not pushed them up.

Reply:
Presenter approach looks much cleaner! 🙌